AsanPardakht.IPG/Abstractions/IServices.cs

```diff
@@ -21,6 +21,18 @@ public interface IServices
         Task<ReverseResponse> Reverse(ReverseRequest data);
         Task<ReverseResponse> Reverse(ulong payGateTranId);
         Task<GenerateTokenResponse> GenerateBuyToken(ulong amountInRials, string callbackURL, string paymentId = null, string mobileNumber = null);
+      /// <summary>
+      /// this methode share amount of payment to accounts
+      /// first you should register IBN in Asan Pardakht and then 
+      /// Asan pardakht do it automatically
+      /// for more info you should talk with AP 
+      /// </summary>
+      /// <param name="amountInRials"></param>
+      /// <param name="callbackURL"></param>
+      /// <param name="paymentId"></param>
+      /// <param name="mobileNumber"></param>
+      /// <returns></returns>
+        Task<GenerateTokenResponse> GenerateBuyDefaultSharingToken(ulong amountInRials, string callbackURL, string paymentId = null, string mobileNumber = null);
```

Comment:
It's better to name the method like this:

```suggestion
        Task<GenerateTokenResponse> GenerateBuyTokenWithDefaultSharing(ulong amountInRials, string callbackURL, string paymentId = null, string mobileNumber = null);
```

Reply:
well its better to not change the method name different from official document. so users can find it with a sample search and so user can match the name with the official document name